guides/15_Background_Filter_Video_Processor.md

```diff
@@ -141,6 +141,18 @@ if (await BackgroundReplacementVideoFrameProcessor.isSupported()) {
 let transformDevice = new DefaultVideoTransformDevice(logger, device, processors);
 ```
 
+### Changing the background image for Background Replacement
```

Comment:
Should we change this to -> Replacing background image mid stream? Also, recommend sophisticating mid stream. Do you mean mid meeting with/without video turned on?

Reply:
Sure, I can clear it up. What I meant by mid stream is anytime after the background replacement processor was created and started processing the video frames.

Reply:
Cool thanks for explanation, would be great to clarify the same for builders, they may run into the same question.